modules/matrix.nix

```diff
@@ -153,6 +160,24 @@ in
       };
 
       virtualHosts = lib.mkMerge [
+        (lib.mkIf cfga.enable {
+          # see https://github.com/etkecc/synapse-admin/blob/main/docs/reverse-proxy.md#nginx
+          "${cfg.domain}" = {
+            forceSSL = lib.mkIf cfg.recommendedDefaults true;
+            locations = {
+              "= /admin".return = "307 /admin/";
+              "^~ /admin/" = {
```

Comment:
Can't this be replaced with a plain location of "/admin/" ?

Reply:
At least in my testing that was always aten by the / from Synapse.